### ISSUE TITLE
Support for react-native-google-analytics-bridge version 6

### ISSUE THREAD

**Issue by kevinsperrine** [rangle/redux-beacon#297]
This is a...
----
 - [ ] :beetle: Bug Report
 - [X] :rocket: Feature Request
 - [ ] :scroll: Documentation Request

**Note:** for support questions, please use [stackoverflow](https://stackoverflow.com/questions/tagged/redux-beacon) with the `redux-beacon` tag. This repository's issues are reserved for feature requests and bug reports.

Which version of Redux Beacon are you using?
----
 - v2.0.3

Which target(s) are you using?
----
 - [ ] Google Analytics
 - [ ] Google Analytics (gtag)
 - [X] React Native Google Analytics
 - [ ] Google Tag Manager
 - [] React Native Google Tag Manager
 - [ ] Amplitude
 - [ ] Segment
 - [ ] Other/Third Party: ...(please specify here)

:rocket: :scroll: What's missing from Redux Beacon that you'd like to add?
----
Following up on https://github.com/rangle/redux-beacon/issues/278#issuecomment-434425286. I didn't realize those hitPayload changes are breaking changes that were added to [react-native-google-analytics-bridge version 6](https://github.com/idehub/react-native-google-analytics-bridge/wiki/Version-6-breaking-changes). Given this, I suppose any changes we embark on here will require bumping the react-native-google-analytics package to version 2.0 already. It seems the best way to support current users and add the new features from v6 of the bridge. Do you agree, @ttmarek?

Can you help out?
----
 - [X] :star2: I am a legend and can get started on a pull request right away given the go-ahead.
 - [ ] :star: I am a superstar and would like to help out given some guidance.
 - [ ] :disappointed: I won't be able to help out on this one.

**Comment by ttmarek** [comment]
I agree @kevinsperrine. Thanks for bringing this up, it's super helpful. It would have completely passed my radar otherwise. 

We should also add the bridge version to the peerDependencies section if you can do that as well. There may be a bit more work here. Feel free to do as much as you can. If you make a pull request with some work I will try to add to it and help out.

I have no problem bumping up to 2.0.
